pkg/kds/mux/client.go

```diff
@@ -282,6 +287,39 @@ func (c *client) startClusters(
 	c.handleProcessingErrors(stream, log, stop, processingErrorsCh, errorCh)
 }
 
+func (c *client) startHealthCheck(
+	ctx context.Context,
+	log logr.Logger,
+	conn *grpc.ClientConn,
+	stop <-chan struct{},
+	errorCh chan error,
+) {
+	client := mesh_proto.NewGlobalKDSServiceClient(conn)
+	log = log.WithValues("rpc", "healthcheck")
+	log.Info("starting")
+
+	go func() {
```

Comment:
we need more logs here

Reply:
where? I'm not sure we should log everytime we make a health check request. should I add a log when the timer duration changes from the last duration?

Reply:
We need to know whether Zone CP sends HC and if this succeeds. We could just log, but if we think it's too much (just like it was with intercp) we can have a metric counter to at least see that it's incrementing.

Additionally, I'd add a log "global cp requested to send healthcheck every X second"  like you pointed out

Reply:
added logs